core/src/main/scala/stryker4s/mutants/applymutants/MatchBuilder.scala

```diff
@@ -20,6 +21,16 @@ class MatchBuilder {
         rest transformOnce {
           case found if found.isEqual(origStatement) && found.pos == origStatement.pos =>
             buildMatch(mutant)
+        } match {
+          case Success(value) => value
+          case Failure(exception) =>
+            error(
+              s"Failed to construct pattern match: original statement [$origStatement] " +
+                s"on line [${origStatement.pos.endLine}] for file [${origStatement.pos.input}].)")
+            error(s"Failed mutation(s) ${mutant.mutantStatements.mkString(",")}.")
```

Comment:
Some interpunction changes I think look nicer: 

```scala
error(s"Failed to construct pattern match. Original statement: [$origStatement]")
error(s"Failed mutation(s) ......")
error(s"at ${file}....")
```

Also, I think we should use the same notation we use when reporting survived mutants [here](https://github.com/stryker-mutator/stryker4s/blob/master/core/src/main/scala/stryker4s/run/report/ConsoleReporter.scala#L30). And with a relativePath. So something like `src/main/scala/stryker4s/Stryker4sRunner:80:14`

I think we should also log that this is likely a user with Stryker4s, and for the user to enable debug logging and create an issue (with link, when in debug?) with the debugged statement

Reply:
We can't get the relative path from the data we currently have sadly. That would require us to pass on the original file as well but I fell that's not the way to go just for logging.